src/Symfony/Component/Routing/Tests/Loader/ObjectLoaderTest.php

```diff
@@ -109,6 +108,13 @@ protected function getObject(string $id): object
     }
 }
 
+class stdClassWithLoadRoutesMethod extends \stdClass
+{
+    public function loadRoutes()
+    {
+    }
+}
```

Comment:
This does not have to be a `stdClass`, does it.

```suggestion
interface CustomRouteLoader
{
    public function loadRoutes();
}
```

Reply:
Updated, it is far more elegant 👍